source/Government.cpp

```diff
@@ -139,6 +139,9 @@ void Government::Load(const DataNode &node)
 			raidFleet = GameData::Fleets().Get(child.Token(1));
 		else if(child.Token(0) == "provoked on scan")
 			provokedOnScan = true;
+		else if(child.Token(0) == "remove" && child.Size() >= 2
+				&& child.Token(1) == "provoked on scan")
+			provokedOnScan = false;
```

Comment:
Can't comment on this myself. @petervdmeer or another maintainer want to take a look at this real quick?

Reply:
Just for transparency, the code is from @Hurleveur , who PR'd it directly to my branch so it could be included here in the same PR it was being used in.

Reply:
Can confirm, I added this because it wasn't possible to remove provoked on scan.

Reply:
Those 3 lines add new `"add/remove"` functionality in a file where none exists yet; so this either introduces a new feature, or it means that the code is at the wrong place. (I didn't review in-depth yet to determine which one it is). The `"bribe"`, `"enforces"`, and `"death sentence"` look like pieces of data that would also benefit from "add/remove", so just adding it as an afterthought in this PR doesn't seem right; please move it to it's own PR.

Issue https://github.com/endless-sky/endless-sky/issues/6341 also has a discussion on making certain outfit-descriptions "conditional". We might want to take the same path here; so add a condition to "provoked on scan" to determine when the provokation happens, like for example a reputation condition).

The data in the datafile could then look like:
```
"provoked on scan"
	or
		"reputation: Remnant" < 100
		"mission escort the top-secret Remnant transport: active"
```
Code that adds new keywords are typically always their own PR, just because the introduction of a new keyword is quite a big thing. Please move this code change to it's own PR and keep it out of this PR.

Reply:
Thanks for reviewing! I'll remove this code from this PR so it can be PR'd and/or handled otherwise elsewhere.

And then I'll need to work out an alternative way to get this desired effect...

Reply:
This PR by Hurleveur https://github.com/endless-sky/endless-sky/pull/6489 adds the desired event mechanics that enable my PR to work. So as requested by Petervdmeer, I'll be bremoving this code from this content PR so it can be handled one way or another via the code PR.

Reply:
I added it in a new PR to get it into the game as fast as possible: https://github.com/endless-sky/endless-sky/pull/7076, as I don't know when that other PR will be merged, and it seems like a blocker to this otherwise pretty much ready PR.

Reply:
Thank you. I appreciate that.

Reply:
Alright, this has been removed now so it can be merged in the stand alone PR #7076 .